source/clear-linux/guides/maintenance/enable-user-space.rst

```diff
@@ -6,22 +6,24 @@ Create and enable a new user space
 This section provides steps to complete the following basic setup tasks for
 a newly installed |CL-ATTR| system:
 
-* Create a new user.
-* Update the OS to its most current version using `swupd`.
-* Install the most common applications for system administrators and
-  developers using bundles.
-* Set up a new user and add the new user to the `wheel` group.
-* Install a GUI to test `sudo` privileges.
+.. contents::
+   :local:
+   :depth: 1
 
-.. note::
-   Log in as the root user to complete the tasks in this
-   section.
+Prerequisites
+*************
+
+Log in as the root user to complete the tasks in this section. Enter:
+
+.. code-block:: bash
+
+   sudo -i
```

Comment:
@mvincerx ok, now this seems out of place as a prerequisite.  Let's no over-complicate this and state that we want the user of this new system to set a root password.

Reply:
Covered in my latest commit! Yes, after current revisions to Finish installation section, this command doesn't make sense.